.env_jfrog

```diff
@@ -0,0 +1,6 @@
+export JF_PRODUCT_DATA_INTERNAL=JF_PRODUCT_DATA_INTERNAL
+export NEWRELIC_LICENSE_KEY=change_me
+export JPD_URL=http://abc.jfrog.io
+export JPD_ADMIN_USERNAME=admin
+export JFROG_ADMIN_TOKEN=change_me
```

Comment:
Not a blocker but I'd like to see a consistent prefix in the JFrog env vars. We have `JF_`, `JPD_`, `JFORG_` prefixes.